### DISCUSSION TITLE
ExecJs not recognizing Nodejs (Docker)

### DISCUSSION THREAD

**Discussion by VascoCorreia** [alshedivat/al-folio#1475] [Q&A]
Hello everybody! 

I am trying to use al-folio with GitHub Pages, following the Getting Started guide. 
My machine uses Windows OS, so I tried using Docker for my local installation.

It seems execJs is not recognizing NodeJs as a Javascript runtime environment. 
I never worked with Ruby, so I do not know how to fix this.
 
Does anyone know what might be the issue?
Below is a picture with a portion of the Log in Docker.

![09 06 2023_16 37 49_REC](https://github.com/alshedivat/al-folio/assets/25254690/b6206c67-2a52-4487-bb6b-1cfff5c19dfe)

**Comment by george-gca** [comment]
The image is barely readable, so it is no use. Is the error still happening?
